systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectS2IResource.java

```diff
@@ -91,8 +91,8 @@ public static KafkaConnectS2I kafkaConnectS2IWithoutWait(KafkaConnectS2I kafkaCo
         return kafkaConnectS2I;
     }
 
-    public static void deleteKafkaConnectS2IWithoutWait(KafkaConnectS2I kafkaConnectS2I) {
-        kafkaConnectS2IClient().inNamespace(ResourceManager.kubeClient().getNamespace()).delete(kafkaConnectS2I);
+    public static void deleteKafkaConnectS2IWithoutWait(String clusterName) {
```

Comment:
I am not sure 100%  but this is `not` clusterName. It is the `Kafka Connect S2I` custom resource name and same with the others. Hmmm?

Reply:
Good question and I can change it to `resourceName` -> this will be better and clear.

Reply:
hmmm `kafkaConnectS2ICrName` would be better and analogically for the others. ResourceName can invoke others that you can use there any type of resource not specifically the KafkaConnectS2I in this case.